modules/ROOT/pages/coverage-maven-concept.adoc

```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-partial:
+:page-deployment-options: acb-ide, studio-ide
```

Comment:
I built this locally, but the pills don't show up. Should they show up?

Reply:
TBH, I don't know if the local build has the property to show the pills. 
The pills appear in your beta site, though ;)